spec.bs

```diff
@@ -853,6 +860,23 @@ To <dfn>fetch script</dfn> given a [=URL=] |url|:
   1. Return |script|.
 </div>
 
+<div algorithm>
+To <dfn>fetch wasm</dfn> given a [=URL=] |url|:
+
+  1. Let |request| be the result of [=creating a request=] with |url|, "`application/wasm`", and
+    null.
+  1. Let |moduleObject| be null.
+  1. [=Fetch=] |request| with [=fetch/processResponse=] set to the following steps given a
+    [=response=] |response|:
+    1. Let |moduleObjectPromise| be the result of [=WebAssembly compileStreaming=] |response|.

```

Comment:
Yes, I had that but removed it by mistake.
We don't need to use [=validate fetching response=], since we only need to check whether response body is null, and check X-Allow-FLEDGE header.
The [[=WebAssembly [compileStreaming=]](https://www.w3.org/TR/wasm-web-api-2/#compile-a-potential-webassembly-response) handles some response checks like the MimeType check.